### PR TITLE
Turn LIGO Plugin from JS to TS

### PR DESCRIPTION
**Pull request by jchenche**

### CODE REVIEW

File: taqueria-plugin-ligo/compile.ts
Comment (mweichert) [inline taqueria-plugin-ligo/compile.ts:4]: @jchenche what is the ts-ignore for?

File: taqueria-plugin-ligo/tsconfig.json
Comment (mweichert) [inline taqueria-plugin-ligo/tsconfig.json:14]: I believe most other packages are targeting ES2021. Do you mind making that adjustment please?

**Review by mweichert — APPROVED** [review]
Looks good! I did add a couple of comments/questions to address before merging, but I'm happy to see this done. Good work Jimmy!

**Review by hu3man — APPROVED** [review]
LGTM aside from the es2021 target version

Approving now so gtm isn't held up

File: taqueria-plugin-ligo/tsconfig.json
Comment (jchenche) [reply taqueria-plugin-ligo/tsconfig.json:14]: will do!

File: taqueria-plugin-ligo/compile.ts
Comment (jchenche) [reply taqueria-plugin-ligo/compile.ts:4]: Without it, it will give me the error: `can only be default-imported using the 'esModuleInterop' flag`

We've seen this before and solved it using `@ts-ignore` due to some 3rd-party library type issue, if I recall correctly.

Comment (mweichert) [reply taqueria-plugin-ligo/compile.ts:4]: Mind trying if `import * as glob from 'fast-glob'` works please?

Comment (jchenche) [reply taqueria-plugin-ligo/compile.ts:4]: Hmm it works though I don't know why the editor is complaining (same version of TS). Can we ignore the complain for VS Code?

Screen Shot 2022-05-02 at 15 14 15

**Comment by hu3man** [comment]
Nice work @jchenche!

File: taqueria-plugin-ligo/compile.ts
Comment (jchenche) [reply taqueria-plugin-ligo/compile.ts]: However, `import glob = require('fast-glob')` works, makes both VS Code and `tsc` happy. Thoughts @mweichert?

**Pull request merged**
